### PR TITLE
Dont error vector nodal test

### PR DESCRIPTION
**Pull request by gridley**
le test (and rebase onto recent changes)

(see last commit)

BTW, I did double-check that the previous error that started all this came up if trying to use the LAGRANGE_VEC magnitude calculator kernel. It indeed does, and your change successfully fixes it.

### CODE REVIEW

File: framework/src/auxkernels/VectorVariableMagnitudeAux.C [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+registerMooseObject("MooseApp", VectorVariableMagnitudeAux);
+
+InputParameters VectorVariableMagnitudeAux::validParams() {
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription("Creates a field consisting of the magnitude of a "
+                             "coupled vector variable.");
+  params.addRequiredCoupledVar(
+      "vector_variable", "The variable from which to compute the component");
+  return params;
+}
```
Comment (lindsayad) [inline framework/src/auxkernels/VectorVariableMagnitudeAux.C:21]: Different formatting. I'm going to get buzzed by `clang-format` but that's fine :smile:

File: test/tests/auxkernels/vector_variable_nodal/tests [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Tests]
+  issues = '#16076'
```
Comment (lindsayad) [inline test/tests/auxkernels/vector_variable_nodal/tests:5]: We'll need a `design` parameter here that references a markdown file. We'll need a `VectorVariableMagnitudeAux.md` file in order to pass documentation testing. You can look at other `.md` files in the `framework/doc/content/source/auxkernels` directory for some examples. Looks like `MaterialRealAux.md` is a fine example. Once you've created `VectorVariableMagnitudeAux.md`, then you can put here
```
issues = '#16076'
design = 'VectorVariableMagnitudeAux.md'
```

**Review by lindsayad — CHANGES_REQUESTED** [review]
This will be a nice contribution. Thanks @gridley . Just the documentation request

File: framework/src/auxkernels/VectorVariableMagnitudeAux.C (hunk 1 above)
Comment (lindsayad) [reply framework/src/auxkernels/VectorVariableMagnitudeAux.C:21]: I'll fix the formatting when this is merged

Comment (gridley) [reply framework/src/auxkernels/VectorVariableMagnitudeAux.C:21]: sounds good; i won't touch this then

File: test/tests/auxkernels/vector_variable_nodal/tests (hunk 2 above)
Comment (gridley) [reply test/tests/auxkernels/vector_variable_nodal/tests:5]: aight, working on that now

**Comment by gridley** [comment]
aight, the docs are in!

**Pull request merged**